src/makeCalculator.js

```diff
@@ -37,7 +37,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  Methods.reset();
```

Comment:
just create an object inside makeCalculator and then return it
this way each time you will call makeCalculator it will create a new object in memory